File: Treinamento/Java/Exercícios_listas/Questão08.java
import java.util.Scanner;
public class Questão08 {
    static void insercaoVetor(int v[]) {
        Scanner s = new Scanner(System.in); 
        for (int c = 0; c < 10; c++) {
            System.out.print("Digite o " + (c + 1) + "° número: ");
            int n = s.nextInt();
            v[c] = n;
        }   
    }
        
    static void impressaoVetor(int v[]) {
        for (int c = 0; c < v.length; c++) {
            System.out.print(v[c] + " ");
        }
        System.out.println();
    }

    static void append(int v[], int n) {
        int c = 0;
        while (v[c] != 0) {
            c++;
        }
        v[c] = n;
    }
 
    public static void main(String[] args) {
        int v1[] = new int[10];
        int v2[] = new int[10];
        int v3[] = new int[20];

        insercaoVetor(v1);
        insercaoVetor(v2);

        for (int c = 0; c < 10; c++) {
            append(v3, v1[c]);           
            append(v3, v2[c]);           
        }
        
        System.out.println("*** Primeiro vetor ***");
        impressaoVetor(v1);
        System.out.println("*** Segundo vetor ***");
        impressaoVetor(v2);
        System.out.println("*** Terceiro vetor ***");
        impressaoVetor(v3);
    }
}

File: Treinamento/Python/Exercícios/Estrutura_de_decisão/Questão08.py
print('*** Ordem dos números ***')
num1 = int(input('Digite o primeiro número: ')) 
num2 = int(input('Digite o segundo número: ')) 
num3 = int(input('Digite o terceiro número: ')) 

if (num1 < num2 and num1 < num3) and num2 < num3:
    print('| ' + str(num3) +', ' + str(num2) + ', '  + str(num1) + ' |')
elif (num2 < num1 and num2 < num3) and num1 < num3:
    print('| ' + str(num3) +', ' + str(num1) + ', '  + str(num2) + ' |')
elif (num3 < num2 and num3 < num1) and num2 < num1:
    print('| ' + str(num1) +', ' + str(num2) + ', '  + str(num3) + ' |')
elif (num3 < num1 and num3 < num2) and num1 < num2:
    print('| ' + str(num2) +', ' + str(num1) + ', '  + str(num3) + ' |')
 
   
    
